hack/hubconfig.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: singapore.open-cluster-management.io/v1alpha1
```

Comment:
You'll need to add to .copyrightignore or change the rule to avoid failing the prow check

Reply:
Added to .copyrightignore. Thanks!